wiki/Article_Style_Guide/en.md

```diff
@@ -1,192 +1,280 @@
-Article Style Guide
-===================
+# Article Style Guide
 
-**Outdated**
+This guide is to serve as a way to help keep consistency in clarity, formatting, and layout between all articles of the osu!wiki.
 
-This article aims to provide a comprehensive style guide for all osu!wiki articles to follow. Although Wikipedia already provides a more general [style guide](https://en.wikipedia.org/wiki/Wikipedia:Manual_of_Style), a more specific one is necessary for *osu!* specific guidelines. As such, only guidelines pertaining to the osu!wiki and its basic formatting rules should be included here.
+All articles should use plain English.
+Your word choice should explain such things in layman's terms (basically, ask yourself, "If the reader is reading this, would they have to lookup any of the words?").
 
-Notability
-----------
+All articles must have proper grammar, correct spelling, and correct information.
+Try your best to do all of those in your pull request and know that reviewers will look for these things before sending it off to the live osu!wiki.
```

Comment:
I will try.

```Markdown
Reviewers may ask for changes in your pull request for blunders or suggestions.
A good osu!wiki writer should read the reviews to improve the quality of writing to ensure for the best for an osu!wiki reader.
```

...Yeah, I went _a bit too_ overboard on that. No idea is that passive enough or too idealistic.